js/activity.js

```diff
@@ -4281,7 +4281,9 @@ function Activity() {
                     }
                 } catch (e) {
                     console.log(e);
-                    textMsg(_("Error: ran out of local storage (file too big)."))
+                    if(e.name === "QuotaExceededError" || e.message === "Not enough space to save locally")
+                        textMsg(_("Error: ran out of local storage (file too big)."));
```

Comment:
Maybe this should say: "Error: Unable to save because you ran out of local storage. Try deleting some saved projects."